### ISSUE TITLE
Decrease runtime of combo indicator

### ISSUE THREAD

**Issue by benjaminysmith** [cmu-delphi/covidcast-indicators#734]
Combo cases and deaths takes 5 hours to run. This needs to be reduced to <1hr before we can deploy it in production.

**Comment by krivard** [comment]
> Andrew  3 minutes ago
has anyone profiled combo cases and deaths? doesn't seem like that indicator does any super heavy lifting, and while it probably calls covidcast more than it needs to, 5 hrs seems far too long

> Katie  1 minute ago
It does need to pull all days all regions of usafacts to be able to catch backfill, but yeah I’m assuming there’s some query-collapsing that can be done

> Katie  < 1 minute ago
it’s currently pulling all regions of jhu too; would be faster to just pull the puerto rico regions

**Comment by chinandrew** [comment]
should be closed by #841. final timing needs to be tested

**Comment by krivard** [comment]
Timing looks good -- from four hours down to twenty minutes.

```
{"elapsed_time_in_seconds":15226.12,"timestamp":"2021-02-20T05:43:48.557537Z"}
{"elapsed_time_in_seconds":15359.83,"timestamp":"2021-02-21T05:46:02.022268Z"}
{"elapsed_time_in_seconds":15286.56,"timestamp":"2021-02-22T05:44:49.319130Z"}
{"elapsed_time_in_seconds":15163.5,"timestamp":"2021-02-23T05:42:45.881870Z"}
{"elapsed_time_in_seconds":15323.56,"timestamp":"2021-02-24T05:45:27.367809Z"}
{"elapsed_time_in_seconds":326.69,"timestamp":"2021-02-24T20:55:15.875932Z"} # (failed job)
{"elapsed_time_in_seconds":15278.35,"timestamp":"2021-02-25T05:44:41.165771Z"}
{"elapsed_time_in_seconds":1156.64,"timestamp":"2021-02-25T16:17:59.430942Z"}
{"elapsed_time_in_seconds":1131.49,"timestamp":"2021-02-26T01:48:54.097485Z"}
{"elapsed_time_in_seconds":1129.28,"timestamp":"2021-02-27T01:48:52.299390Z"}
{"elapsed_time_in_seconds":1127.3,"timestamp":"2021-02-28T01:48:49.578366Z"}
{"elapsed_time_in_seconds":1129.23,"timestamp":"2021-03-01T01:48:51.366035Z"}
```
